main.cpp

```diff
@@ -545,14 +431,126 @@ MOVE THEM to the space below this block comment and put them in numerical order
     simply CUT and PASTE them in the space provided below:
 */
 
+/*
+Thing 1) MIDI Keyboard
+5 properties:
+    1) number of keys (int)
+    2) number of knobs (int)
+    3) number of sliders (int)
+    4) number of buttons (int)
+    5) number of pedals (int)
```

Comment:
everything is an integer.
what about 'brand name' or 'model name' or something else that can be represented with something other than an integer?
- [ ] revise